src/Swarm/TUI/View/Objective.hs

```diff
@@ -31,19 +33,25 @@ makeListWidget (GoalTracking _announcements categorizedObjs) =
   objList = intercalate [Spacer] $ map f $ M.toList categorizedObjs
   f (h, xs) = Header h : map (Goal h) (NE.toList xs)
 
-renderGoalsDisplay :: GoalDisplay -> Widget Name
-renderGoalsDisplay gd =
-  if hasMultiple
-    then
-      vBox
-        [ hBox
-            [ leftSide
-            , padLeft (Pad 2) goalElaboration
-            ]
-        , footer
-        ]
-    else goalElaboration
+renderGoalsDisplay :: GoalDisplay -> Maybe (Document Syntax) -> Widget Name
+renderGoalsDisplay gd desc =
+  vBox
+    [ maybe emptyWidget (hCenter . padTop (Pad 1) . withAttr boldAttr . drawMarkdown) desc
+    , goalsWidget
+    ]
  where
+  goalsWidget =
+    if hasMultiple
+      then
+        vBox
+          [ hBox
+              [ leftSide
+              , padLeft (Pad 2) goalElaboration
+              ]
+          , footer
+          ]
+      else goalElaboration
```

Comment:
```suggestion
  goalsWidget
    | hasMultiple =
          vBox
          [ hBox
              [ leftSide
              , padLeft (Pad 2) goalElaboration
              ]
          , footer
          ]
    | otherwise = goalElaboration
```
I find `if-then-else` ugly, and this seems an easy place to get rid of it.